templates/referent/users_list.html.twig

```diff
@@ -36,6 +36,18 @@
                 <div class="manager__filters__row">
                     <div class="manager__filters__section">
                         <div class="manager__filters__group">
+                            {% dump  managedArea.tags.count %}
+                            {% if managedArea.tags.count > 1 %}
```

Comment:
what about 
```twig 
{% set tags = app.user.managedArea.tags %}
{% if tags.count > 1 %}
...
```

Reply:
Not possible, because this page is also accessible by a co-referent and `app.user.managedArea.tags` can not be used.